Before: 78f3b619cc8306904e4788b0b9b5eeacff43599c65541c88ee56de368934b490
After: b47a25500cb2540489b9f3498f62edfb607e4c1118e8ab5ef82b2f949c652ce4
Patch: 8fa40e58e562291b2a4970692e5e87cfcb8b0dc220e6d4d153c985efc15b4f79
--- .yardopts	2012-07-22 20:44:49.365583426 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,5 +0,0 @@
---db         target/yardoc.db
---output-dir target/doc
---markup     markdown
---readme     README.md
-{app,lib}/**/*.rb
--- Gemfile	2012-07-22 20:44:49.365583426 +1000
+++ Gemfile	2012-07-22 21:28:05.356852111 +1000
@@ -1,8 +1,5 @@
 source :rubygems
 
 # Load Gemfile.corvid
-eval File.read(__FILE__ + '.corvid')
-
-# Parser for Markdown documentation
-gem 'rdiscount', group: :doc
+eval File.read(File.expand_path('../Gemfile.corvid',__FILE__))
 
--- Gemfile.corvid	2012-07-22 20:44:49.365583426 +1000
+++ Gemfile.corvid	2012-07-22 21:28:05.356852111 +1000
@@ -17,7 +17,6 @@
 group :test_unit do
   gem 'minitest'
   gem 'guard-minitest', '>= 0.5.0', require: false
-  gem 'turn'
 end if Dir.exists?('test/unit')
 
 group :test_spec do
@@ -27,6 +26,7 @@
 
 group :doc do
   gem 'yard'
+  gem 'rdiscount' # TODO aaaaaaaaaahhhhhhhhhh
 end
 
 group :debug do
--- README.md	2012-07-22 20:44:49.365583426 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,6 +0,0 @@
-# My Project [![Build Status](https://secure.travis-ci.org/myusername/myproject.png?branch=master)](http://travis-ci.org/myusername/myproject) [![Dependency Status](https://gemnasium.com/myusername/myproject.png)](https://gemnasium.com/myusername/myproject)
-
-For the above Travis CI and Gemnasium buttons to work you must replace:
-* `myusername` with your Github username.
-* `myproject` with your project's name on Github.
-
--- bin/yard	2012-07-22 20:44:49.365583426 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,16 +0,0 @@
-#!/usr/bin/env ruby
-#
-# This file was generated by Bundler.
-#
-# The application 'yard' is installed as part of a gem, and
-# this file is here to facilitate running it.
-#
-
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
-
-require 'rubygems'
-require 'bundler/setup'
-
-load Gem.bin_path('yard', 'yard')
--- bin/yardoc	2012-07-22 20:44:49.365583426 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,16 +0,0 @@
-#!/usr/bin/env ruby
-#
-# This file was generated by Bundler.
-#
-# The application 'yardoc' is installed as part of a gem, and
-# this file is here to facilitate running it.
-#
-
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
-
-require 'rubygems'
-require 'bundler/setup'
-
-load Gem.bin_path('yard', 'yardoc')
--- bin/yri	2012-07-22 20:44:49.365583426 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,16 +0,0 @@
-#!/usr/bin/env ruby
-#
-# This file was generated by Bundler.
-#
-# The application 'yri' is installed as part of a gem, and
-# this file is here to facilitate running it.
-#
-
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
-
-require 'rubygems'
-require 'bundler/setup'
-
-load Gem.bin_path('yard', 'yri')
